docs/configuration/routers/register.md

```diff
@@ -1,9 +1,26 @@
 # Register routes
```

Comment:
We should revert those changes. (except grammar fixes of courses)

Reply:
Yep, agreed